frontend/src/components/checklist/ChecklistItemTree.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <table style="width: 100%; border-spacing: 0" :style="getRowStyle()">
+    <tr>
+      <td colspan="2" style="text-wrap: wrap; line-height: 36px; padding-left: 6px">
+        {{ getPositionNumber(checklistItem) }})
+        {{ checklistItem.text }}
+      </td>
+      <td style="max-width: 400px; text-align: right">
+        <div
+          v-for="activity in getActivities(checklistItem)"
+          :key="activity._meta.self"
+          style="display: inline; text-wrap: nowrap; padding-right: 4px"
+        >
+          <ScheduleEntryLinks :activity-promise="activity._meta.load" />
+        </div>
+      </td>
+      <td style="width: 30px">
+        <v-btn
+          v-if="getActivities(checklistItem).length > 0"
+          icon
+          @click="copyToClipboard(checklistItem)"
+        >
+          <v-icon>mdi-content-copy</v-icon>
+        </v-btn>
+      </td>
+    </tr>
+    <tr
+      v-for="subItem in sortBy(checklistItem.children().items, (c) => c.position)"
+      :key="subItem._meta.self"
+    >
+      <td style="width: 20px"></td>
+      <td colspan="3">
+        <ChecklistItemTree :checklist-item="subItem" />
+      </td>
+    </tr>
+  </table>
+</template>
+
+<script>
+import { sortBy } from 'lodash'
+import ScheduleEntryLinks from '../material/ScheduleEntryLinks.vue'
+
+export default {
+  name: 'ChecklistItemTree',
+  components: {
+    ScheduleEntryLinks,
+  },
+  props: {
+    checklistItem: { type: Object, required: true },
+  },
+
+  methods: {
+    getActivities(checklistItem) {
+      const camp = checklistItem.checklist().camp()
+      const activities = camp.activities().items
+      const checklistNodes = checklistItem.checklistNodes().items
+
+      return activities.filter((a) =>
+        checklistNodes.some((cn) => cn.root().id == a.rootContentNode().id)
+      )
+    },
+    getRowStyle() {
+      const globalPos = this.getTotalNumberOfItemsAbove(this.checklistItem)
+      return globalPos % 2 == 0
+        ? { backgroundColor: '#EEE' }
+        : { backgroundColor: '#DDD' }
+    },
+
+    getTotalNumberOfItemsAbove(checklistItem) {
```

Comment:
i have the feeling, parts of this can be implemented with computed methods (which update when the dependencies update)